sphinx/writers/html.py

```diff
@@ -687,6 +690,15 @@ def visit_math(self, node, math_env=''):
                           (self.builder.current_docname, node.line))
         raise nodes.SkipNode
 
+    # overwritten to do not add '</dt>' in 'visit_definition' state.
+    def visit_definition(self, node):
+        self.body.append(self.starttag(node, 'dd', ''))
+        self.set_first_last(node)
+
+    # overwritten to add '</dt>' in 'depart_term' state.
+    def depart_term(self, node):
+        self.body.append('</dt>\n')
```

Comment:
Overriding depart-method is a little strange to me. It makes strong coupling. 
It would be nice to override the pair of visitor and departure.